src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1685,26 +1645,7 @@ void exit() {
      */
     @Deprecated(since="1.2", forRemoval=true)
     public final void stop() {
-        @SuppressWarnings("removal")
-        SecurityManager security = System.getSecurityManager();
-        if (security != null) {
-            checkAccess();
-            if (this != Thread.currentThread()) {
-                security.checkPermission(SecurityConstants.STOP_THREAD_PERMISSION);
-            }
-        }
-
-        if (isVirtual())
-            throw new UnsupportedOperationException();
-
-        // A zero status value corresponds to "NEW", it can't change to
-        // not-NEW because we hold the lock.
-        if (started && (eetop != NO_REF)) {
-            resume(); // Wake up thread if it was suspended; no-op otherwise
-        }
-
-        // The VM can handle all thread states
-        stop0(new ThreadDeath());
```

Comment:
This is the time to remove `stop0()` and `stopImpl()`.

Reply:
Removed `stop0()`, `stopImpl()` and `stopCalled`.
Created OpenJ9 PR https://github.com/eclipse-openj9/openj9/pull/15963

Reply:
Thanks, I had missed that `stopCalled` could be removed.